devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -188,9 +188,20 @@ private List<Map> handleAcceleration(descMap) {
 
 private List<Map> parseAxis(List<Map> attrData) {
 	def results = []
-	def x = hexToSignedInt(attrData.find { it.attrInt == 0x0012 }?.value)
-	def y = hexToSignedInt(attrData.find { it.attrInt == 0x0013 }?.value)
-	def z = hexToSignedInt(attrData.find { it.attrInt == 0x0014 }?.value)
+
+	def attrsFound = []
+	for (attrId in [0x0012, 0x0013, 0x0014]) {
```

Comment:
Yeah we just need to defer the calling of `hexToSignedInt` until after that check.